.github/workflows/test.yml

```diff
@@ -538,7 +538,7 @@ jobs:
             '*.spec.js',
             'application/**/*.spec.js',
             'checker/**/*.spec.js',
-            'clone/**/*.spec.js',
+            'clone/*.spec.js',
```

Comment:
are we happy with this only getting the tests on the root of this folder

Reply:
I think so - there arent any fed-disabled tests here (I could add a test that v0 goes to v0 but feels unnecessary given the FF is being removed soon)